Makefile

```diff
@@ -1,52 +1,28 @@
 start: app
 
-test:
-	make testunit
-
-testunit: composer rmTestDb upTestDb yiimigratetestDb yiimigratetestDblocal rmTestDb
-	docker-compose run --rm cli bash -c 'MYSQL_HOST=testDb MYSQL_DATABASE=test ./vendor/bin/codecept run unit'
-
-app: upDb composer yiimigrate yiimigratelocal
+app: db deps tables basemodels
 	docker-compose up -d app
 
-composer:
-	docker-compose run --rm --user="0:0" cli composer install
+deps:
+	docker-compose run --rm cli composer install
 
-composerupdate:
-	docker-compose run --rm --user="0:0" cli composer update
+depsupdate:
+	docker-compose run --rm cli composer update
 
-rmDb:
-	docker-compose kill db
-	docker-compose rm -f db
+depsrefresh: depsupdate deps
```

Comment:
Unless I'm mistaken, running `composer update` both updates and installs the dependencies, so I don't think `depsrefresh` is necessary.